### PR TITLE
Add `theme-darcula` for JupyterLab

### PR DESCRIPTION
**Pull request by dhirschfeld**
Checklist
- [x] Title of this PR is meaningful: e.g. "Adding my_nifty_package", not "updated meta.yaml".
- [x] License file is packaged (see [here](https://github.com/conda-forge/staged-recipes/blob/master/recipes/example/meta.yaml#L57-L66) for an example).
- [x] Source is from official source.
- [x] Package does not vendor other packages. (If a package uses the source of another package, they should be separate packages or the licenses of all packages need to be packaged).
- [x] If static libraries are linked in, the license of the static library is packaged.
- [x] Build number is 0.
- [x] A tarball (`url`) rather than a repo (e.g. `git_url`) is used in your recipe (see [here](https://conda-forge.org/docs/maintainer/adding_pkgs.html#build-from-tarballs-not-repos) for more details).
- [x] GitHub users listed in the maintainer section have posted a comment confirming they are willing to be listed there.
- [ ] When in trouble, please check our [knowledge base documentation](https://conda-forge.org/docs/maintainer/knowledge_base.html) before pinging a team.

### CODE REVIEW

**Comment by dhirschfeld** [comment]
```
 `jlpm` unavailable. 
```

**Comment by dhirschfeld** [comment]
@telamonian let me know if you'd like to be a maintainer here. No pressure to do so - I'm happy to manage the `conda` package.

**Comment by dhirschfeld** [comment]
ping @conda-forge/staged-recipes - I think this is good to review/merge!

**Review by mariusvniekerk — APPROVED** [review]

**Comment by wolfv** [comment]
I am disappointed that they're not namespacing this theme with `jupyterlab-theme-darcula`. But since it's the name on pipy I guess we should stick with it.

File: recipes/theme-darcula/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,46 @@
...
+    - nodejs
+    - python >=3.6
+    - pip
+    - setuptools >=40.8.0
+  run:
+    - jupyterlab >=3.0.0,<4
+    - python >=3.6
+
+test:
+  commands:
```
Comment (wolfv) [inline recipes/theme-darcula/meta.yaml:30]: there are two `commands` keys here? That smells like invalid yaml :)

Comment (dhirschfeld) [reply recipes/theme-darcula/meta.yaml]: Surprised it passed both the linter and parsing! I removed the dummy key.

**Pull request merged**